### DISCUSSION TITLE
Review your docs

### DISCUSSION THREAD

**Discussion by stratboy** [spatie/laravel-medialibrary#3517] [Ideas]
Hi, it would be great if official docs included tech docs about at least the most used library methods, as well as explicit install/uninstall instructions.

**Comment by freekmurze** [comment]
There are installation instructions in our docs, and examples of the most common operations. Uninstalling isn't covered, but should be clear for people with some experience using PHP / Laravel (remove package from composer.json, remove db tables / files, remove traits from your model (the reverse steps from installation, really))
